Runner/suites/Kernel/FunctionalArea/baseport/USBHost/run.sh

```diff
@@ -1,6 +1,8 @@
 # Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
 # SPDX-License-Identifier: BSD-3-Clause-Clear
 
+#Setup requires at least one USB peripheral connected to USB port that supports Host mode function
```

Comment:
@achauras-qcom could you just add it to the first commit in the series? No need to create a separate commit for it.